bpy-build.yaml

```diff
@@ -0,0 +1,13 @@
+addon_folder: MCprep_addon
+build_name: MCprep_addon
+
+install_versions:
+  - '3.5'
+  - '3.4'
+
+during_build:
+  default:
+    - [flake8 --extend-ignore W191 .]
+  dev:
+    - create_file("mcprep_dev.txt")
```

Comment:
What does this do? Just creates the empty file, for the addon to recognize run as a dev build?

Reply:
Yep, it just creates an empty file

Reply:
Might be good to add a yaml comment inline which clarifies why we do this. Maybe:

`# Create dev file to force dev tracking and other behavior`